htdocs/product/stock/movement_list.php

```diff
@@ -66,8 +66,8 @@
 $result = restrictedArea($user, 'stock');
 
 $idproduct = GETPOST('idproduct', 'int');
-$year = GETPOST("year");
-$month = GETPOST("month");
+$search_date_start = dol_mktime(0, 0, 0, GETPOST('search_date_startmonth', 'int'), GETPOST('search_date_startday', 'int'), GETPOST('search_date_startyear', 'int'));
```

Comment:
Because date is an input of user the last parameter of dol_mktime here must be 'tzuserrel'

Reply:
I add 'tzuserrel'